src/Layout/NavBarMenu/NavBarMenu.jsx

```diff
@@ -91,8 +91,8 @@ class NavBarMenu extends PureComponent {
     let uid = this.props.firebase.auth().currentUser.uid;
     let remove = {};
     remove[
-      ["sensors." + sensor]
-    ] = this.props.firebase.firestore.FieldValue.delete();
+      [ "sensors." + sensor ]

```

Comment:
Just leaving a comment here since I know you didn't write this code but it's got a needless array here; I'll fix it afterwards. It evaluates to pretty much the same thing anyway since it'll first convert the array to a string before hashing it to determine the key.

Reply:
Referenced in #9.